src/components/Button/Button.tsx

```diff
@@ -40,10 +49,12 @@ const Button = ({
   return (
     <button
       type="button"
-      className={["storybook-button", `storybook-button--${size}`, mode].join(
+      className={["storybook-button",
```

Comment:
btw dont'use storybook button, the point is this button is not used for storybook purposes, storybook is just a way to display the button but not the end of it